src/st_stuff.c

```diff
@@ -769,11 +771,22 @@ void ST_updateWidgets(void)
           switch (plyr->keyblinkkeys[i])
           {
             case KEYBLINK_EITHER:
-              if ( (plyr->keyblinktics & (2*KEYBLINKMASK)) &&
-                  !(plyr->keyblinktics & (4*KEYBLINKMASK)))
+              if (st_keyorskull[i] == KEYBLINK_SKULL)
```

Comment:
Please apply the same logic to the Boom HUD widget, too.

Reply:
Or maybe you could even factor this out and share the code between both occurances.

Reply:
Oops, I forgot. Done in 6efaa208d833b51820c4733bc9b57395444ecf70.

Reply:
> Or maybe you could even factor this out and share the code between both occurances.

I'll see what I can do, later.

Reply:
Alright, how's 90cc8f10bef2624f5dc6c541adca80b2023a4892?

Reply:
> Alright, how's [90cc8f1](https://github.com/fabiangreffrath/woof/commit/90cc8f10bef2624f5dc6c541adca80b2023a4892)?

Beautiful, thank you!